app/models/vm_or_template.rb

```diff
@@ -1833,6 +1837,14 @@ def parent_resource
     parent
   end
 
+  def available_child_options
+    Rbac.filtered(VmOrTemplate.active.by_ems.where.not(:id => id).select { |vm| vm.parent.nil? })
```

Comment:
Why `active` ? Where it was in previous UI code ?

Reply:
Active is the scope that says that the ems has to be present.